develop/README.md

```diff
@@ -0,0 +1,69 @@
+# Local Development Environment
+
+The scripts and manifests in this folder assist in creating a development environment for rapid iteration on `deployment-validation-operator`. Scripts for installing and bootstrapping the environment are included.
+
+Features:
+
+- Multi-node Kubernetes cluster in Docker with [K3D](https://k3d.io/)
+- Local image registry server
+- On golang / operator code changes, trigger automated container builds + Kubernetes workload updates with [Tilt](https://tilt.dev/)
+
+## Usage
+
+### Install & Configure
+
+1. Run install
+
+`$ bash install.sh`
```

Comment:
should maybe say:

Run install from the deployment-validation-operator root directory:
`$ bash develop/install.sh`